src/user-management/user-details/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>User Management</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="constants.js" defer></script>
+    <script src="utils.js" defer></script>
+    <script src="script.js" defer></script>
+  </head>
+  <body>
+    <header class="header">
+      <h1>User Management</h1>
+    </header>
+    <section id="main-section">
+      <div class="user-details">
+        <div class="user-details__personal"></div>
+        <ul class="user-details__accordion hide accordion">
+          <li class="accordion__academic">
+            <div class="visible-content">
+              <h2>Academics or Professional Details</h2>
+              <img src="../images/arrow-icon.svg" alt="expand" />
+            </div>
+            <div class="hidden-content hide"></div>
+          </li>
+          <li class="accordion__tasks">
+            <div class="visible-content">
+              <h2>Tasks</h2>
+              <img src="../images/arrow-icon.svg" alt="expand" />
+            </div>
+            <div class="hidden-content hide"></div>
```

Comment:
Do we need this div as they don't contain anything and the content is coming from javascript?